### PR TITLE
Cult Item Tweaks and a new addition: The cult BASTARD SWORD

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
add: Blood cultists can now create a unique bastard sword at their forge
del: Blood cultists can no longer obtain a hardsuit from the forge
balance: Shielded robes now only hold 1 charge, down from 3, but will be able to regenerate
/:cl:

So I've been thinking about cult items and while most managed to find their niche, the hardsuit has just been lacking. It servers little purpose beyond being a space suit and cultists that even set up in areas exposed to space are better off just dropping a few pylons than trying to get everyone in the cult to get a space suit. 

I've been wanting to add a weapon to the forge for quite some time but I could never quite think of an appropriate addition. All my ideas were not unique at all and basically boiled down to reskins of existing weapons. UNTIL NOW.

The cult bastard sword (name pending) is an extremely powerful 40-force weapon with a 50% block rate that can only be held with 2 hands and cannot be pulled or placed in any inventory slot. It uses a BIGHUGE 64x64 icon as well. To avoid RUSH THE BRIG memes and greedy cultists revealing their presence early, this weapon can only be forged after about 20 minutes into the round. 

What makes this weapon unique are three special abilities:

1) Using Kor's dash code, the user can jaunt within their visual range (Note: This isn't as crazy as it sounds since your mouse will be out of position for a melee attack immediately following the jaunt). 

2) The sword acts as a soul reservoir. Upon attacking any crit or dead target, the sword will absorb their soul like a soulstone. The key difference is that the souls cannot be released as shades and can only be used on construct shells. 

3) The sword also has an action ability called Geometer's Fury (name pending) that empowers the user with a whirlwind-type attack for 5 seconds. The ability will slow down the user but make them effectively invincible during this period. Obvious audio and visual cues will make the ability easy to notice when used in combat. During the whirlwind the sword will attack all adjacent targets at the same rate that normal melee attacks occur.

The abilities are currently on a 25s cooldown.

### CODE REVIEW

**Comment by lzimann** [comment]
Freeze

**Comment by juanototo2** [comment]
Robustin? That bastard sword's main ability looks like a clone of wc3 blademaster, but it's not that much of a problem, and I think melee aimbot can't be worse than portable gangchanka. Let's test it and try not to turn every single hallway into a 3 hit crit meat grinder.

If you can, add a admin variant of this piece of shit.

**Comment by Repukan** [comment]
We Juggernaut now?

File: code/game/gamemodes/cult/cult_items.dm [hunk 1]
```diff
@@ -63,6 +63,174 @@
...
+	item_state = "cultbastard"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	actions_types = list()
+	var/datum/action/innate/spin2win/linked_action
+	var/spinning = FALSE
+	var/dash_toggled = FALSE
```
Comment (KorPhaeron) [inline code/game/gamemodes/cult/cult_items.dm]: I was gonna pull this weapon into my PR but it looks like you just straight copypasted all the code so it wouldn't pass review anyway

**Comment by Robustin** [comment]
@KorPhaeron 

This must be the first time you've ready my code then. Yes I copied your dash stuff and made some appropriate changes with the intention of changing this to a dash subtype once your stuff actually got merged. It was a small part of this PR though, the "no pull" objects, the whirlwind, the weaponized soulstone, getting all these effects to play nice under the same weapon wasn't easy and it went through a good deal of testing.

**Pull request closed without merging**